softwareComponents/kinematics/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+cmake_minimum_required(VERSION 3.6)
+project(rofi)
+
+set(CMAKE_CXX_STANDARD 17)
+
+if(${LIBCXX})
```

Comment:
Tento if (když už) patří do suite, nikoliv do projektu. Pokud je tvůj program přeložen s libc++, ale zbytek s libstdc++, tak se to šeredně rozbije.